db/src/main/java/org/trellisldp/ext/db/DBUtils.java

```diff
@@ -65,6 +67,14 @@ public static BinaryMetadata getBinaryMetadata(final IRI ixnModel, final String
         return null;
     }
 
+    public static boolean isLdpTypeTriple(final Triple triple, final IRI identifier) {
```

Comment:
Can this be put into a util class in the main repo reused here.

Reply:
That's a good idea.